solutions/mint-nft/hardhat/artifacts/@openzeppelin/contracts/token/ERC721/ERC721.sol/ERC721.dbg.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Are all these ( NFT contract deployment ) necessary and the entire artifacts folder? - We would like to keep the example as minimal as possible for easy adoption

Reply:
So different to prior implementation that use lazy minting, this requires on-chain minting and will need those artifacts folder, while they can be generated on the fly, they are generally pushed (not put in .gitignore) to the repo so the ABI can be used in production to communicate with the deployed smart contract on the client side

As far as implementation this should be pretty simple, artifacts might look complicated but most of the time it is unnecessary for user to really know what's inside (kinda like `node_modules`), they just need to know that the ABI is there and they can search for the ABI when they need them